WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -815,6 +812,30 @@ private void LogCoinJoinSummary(ImmutableArray<AliceClient> registeredAliceClien
 			}
 		}
 
+		double winnerCost = 0;
+		while ((winner.Sum(x => x.Amount) > liquidityClue) && (winnerCost/winner.Sum(x => x.Amount) < 3))
```

Comment:
3 is a magic number here, this I'll test and figure out very well after we have the logs: https://github.com/zkSNACKs/WalletWasabi/pull/8938/files#r948849346